realm/realm-library/src/syncIntegrationTest/java/io/realm/SyncedRealmIntegrationTests.java

```diff
@@ -276,4 +276,21 @@ public void waitForInitialRemoteData_readOnlyFalse_upgradeSchema() {
             user.logout();
         }
     }
+
+    @Test
+    public void defaultRealm() throws InterruptedException {
+        SyncCredentials credentials = SyncCredentials.nickname("test", true);
+        SyncUser user = SyncUser.login(credentials, Constants.AUTH_URL);
+        SyncConfiguration config = SyncConfiguration.automatic();
+        Realm realm = Realm.getInstance(config);
+        SyncManager.getSession(config).downloadAllServerChanges();
+        realm.refresh();
+
+        try {
+            assertFalse(realm.isEmpty());
```

Comment:
I assume it's `false` because you expect the Permission schema to be populated?

Reply:
Yes, I need to merge in your branch to verify though